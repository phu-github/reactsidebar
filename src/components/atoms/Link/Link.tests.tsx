import React from 'react';
import renderer from 'react-test-renderer';
import {Link} from '.';

describe('Link component tests', () => {
  it('Should render correctly with default props', () => {
    const component = renderer.create(<Link text="Session start" url="http://www.google.com">Test Link</Link>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
