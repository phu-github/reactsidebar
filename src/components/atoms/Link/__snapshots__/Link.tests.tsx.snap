// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component tests Should render correctly with default props 1`] = `
<a
  className="link-style"
  href="http://www.google.com"
  style={
    Object {
      "color": "#007FA1",
      "fontSize": undefined,
    }
  }
>
  Session start
  Test Link
</a>
`;
